### PR TITLE
Feat: add --all-clusters flag for command addon disable

### PR DESCRIPTION
**Pull request by ycyaoxdu**
for this comment https://github.com/open-cluster-management-io/clusteradm/pull/102#discussion_r790763476

### CODE REVIEW

File: pkg/cmd/addon/disable/exec.go [hunk 1]
```diff
@@ -61,6 +60,25 @@ func (o *Options) run() (err error) {
...
+			return err
+		}
+		for _, item := range mcllist.Items {
+			clusters.Insert(item.ObjectMeta.Name)
+		}
+	} else {
+		clusters = sets.NewString(o.clusters...)
+	}
+
+	klog.V(3).InfoS("values:", "addon", addons, "clusters", clusters)
```
Comment (qiujian16) [inline pkg/cmd/addon/disable/exec.go:80]: this log message seems not quite readable :)

File: pkg/cmd/addon/disable/cmd.go [hunk 2]
```diff
@@ -47,6 +47,7 @@ func NewCmd(clusteradmFlags *genericclioptionsclusteradm.ClusteradmFlags, stream
 
 	cmd.Flags().StringSliceVar(&o.names, "name", []string{}, "Names of the add-on to deploy (comma separated)")
 	cmd.Flags().StringSliceVar(&o.clusters, "cluster", []string{}, "Names of the managed cluster to deploy the add-on to (comma separated)")
+	cmd.Flags().BoolVar(&o.allclusters, "all-clusters", false, "Make all maanged clusters to disable the add-on")
```
Comment (qiujian16) [inline pkg/cmd/addon/disable/cmd.go:50]: type maanged

otherwise looks good

/approve

Comment (ycyaoxdu) [reply pkg/cmd/addon/disable/cmd.go]: so sorry for my caleless...edited it and the examples https://github.com/open-cluster-management-io/clusteradm/pull/115/files#diff-ce8c5a42db5d6eaba36ade94a6c4420c6c152bd06a92e3489f306e3e0db56624R15-R18

**Review by qiujian16 — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**